README.md

```diff
@@ -224,7 +224,7 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 #### **OpenEBS**
 
 - **Swag**: Stickers, T-shirt, Laptop.
-- **Requirements**: Contribute to either [openEBS](https://github.com/openebs/openebs/labels/Hacktoberfest) or [Litmus](https://github.com/litmuschaos/litmus/labels/Hacktoberfest)
+- **Requirements**: Contribute to either [openEBS](https://github.com/openebs/openebs/labels/Hacktoberfest) or [Litmus](https://github.com/litmuschaos/litmus/labels/Hacktoberfest), the bset PR submission will win a Laptop.
```

Comment:
Thanks for the clarification here. Please check the spelling of "best" and would you be able to add OpenEBS's laptop giveaway to the **Top Contributors** section as well? :+1:

Reply:
Thanks for the reminder!
I've fixex the typo and add it to **Top Contributors** section